main.py

```diff
@@ -42,21 +40,48 @@ def start_tornado_server() -> None:
     logger.info('Server shut down, exiting...')
 
 
-def start_rabbit_mq():
-    logger.info('Started rabbit mq')
+def message_callback(message):
+    # number of workers should be configurable
+    max_number_of_worker = 2
+    worker = Worker(message)
+    worker.proceed_message()
+
+
+def create_queue_adaptor():
+    # value hardcoded temporary, should be taken from config or/and secret config
+    return proton_queue_adaptor.ProtonQueueAdaptor(
+        urls=['amqp://localhost:5672'],
+        queue='nhais_inbound',
+        username='quest',
+        password='quest',
+        max_retries='3',
+        retry_delay=0.1,
+        get_message_callback=message_callback)
+
+async def send_message_to_queue(adaptor):
+    #temporary code, to be replaced by proper tests
```

Comment:
the sending does not work now, this method is only for test/debug purposes and should be removed